docs/pages/Configuration.md

```diff
@@ -750,6 +751,12 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.2
+
+**Features:**
+
+- Adds a new bundling mode `depCache` for providing a dependencies cache: [`builder.bundles.bundleDefinition.sections.mode=depCache`](#properties)
+
```

Comment:
For previous versions we had also a line saying e.g. `Specification Version 3.0 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli) v3.0.0 and above.`.

This seems to be missing for 3.1 and 3.2.

Reply:
Done